app/assets/stylesheets/reviews.scss

```diff
@@ -1,3 +1,5 @@
+@import "variables";
```

Comment:
should not be using `require` directives in application.scss - shift to `import` directives, and make sure `variables.scss` is imported before `reviews.scss`